### DISCUSSION TITLE
Is Svelte feature parity planned?

### DISCUSSION THREAD

**Discussion by yekta** [inokawa/virtua#477] [Q&A] [answered]
We are looking to use the virtual list for a messaging like list. However Svelte version seems to be missing a couple of things to achieve it (most important one being reverse param). Is this planned in near feature?

Note: The zero-config approach is awesome. This is the first virtualizer I've used that "just works".
Note 2: Manually adding that behaviour seems easy enough I realized.

**Comment by inokawa** [comment, marked as answer]
Hello.
All of React/Vue/Solid/Svelte adaptors are built with shared logic and almost same design. So in theory, we can port any features of React adaptor to Svelte adaptor.
I'm not a Svelte expert so it might be late. And of course PR welcome.
